content/posts/newsletter/2019-11-08.md

```diff
@@ -0,0 +1,125 @@
+---
+title: XMPP Newsletter, 08 Nov 2019, Sprints, IoT, and early Twitter
+save_as: 2019/11/newsletter-08-novembre/index.html
+url: 2019/11/newsletter-08-november/
+date: 2019-11-08 13:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter covering the month of October.
+
+This is a community effort, and [the process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter): please help us spread the word, share this newsletter.
+
+## Articles
+
+QuickBlox has written a blog post about [the use XMPP in 2019](https://quickblox.com/blog/why-quickblox-choose-xmpp-protocol/).
+
+![Quickblox examples](https://quickblox.com/wp-content/uploads/2019/10/xmpp-body.png)
+
+Martin aka debacle wrote an short piece bout [Dino and other updated software](https://xmpp-team.pages.debian.net/blog/2019/10/dino-with-jet-and-bookmarks2.html) in Debian.
+
+Ben Kwiecien [compared ejabberd and Prosody](https://kwiecien.us/archives/2019/10/16/ejabberd_vs__prosody/index.html) in plain text.
+
+Neetesh Mehrotra wrote about XMPP as a [Communication Protocol for the IoT](https://opensourceforu.com/2019/10/xmpp-a-communication-protocol-for-the-iot/), comparing it to MQTT and HTTP.
+
+Jon Erlichman published a photo of an early sktech of Twitter in 2005 by Jack Dorsey... which features [Jabber at the core](https://twitter.com/JonErlichman/status/1191369676531621888).
+
+![Jack Dorsey's 2005 Twitter](https://pbs.twimg.com/media/EIiYZ8PXkAYuf5Y?format=jpg&name=900x900)
+
+And last but not least, there are translations everywhere! The newsletter for the last month has been translated in [German](https://www.jabber.de/xmpp-newsletter-01-oktober-2019-fosdem-2020-modernisierung-von-xmpp-peer-networks/) and [Spanish](https://www.jabber.de/xmpp-newsletter-01-de-octubre-de-2019-fosdem-2020-modernizacion-de-xmpp-peer-networks/)!
+
+## Tutorials
+
+Erlang Solutions republished their tutorial on how to [Build a complete iOS messaging app using XMPPframework](https://www.erlang-solutions.com/blog/build-a-complete-ios-messaging-app-using-xmppframework-tutorial-part-1.html), with its [part 2](https://www.erlang-solutions.com/blog/xmpp-protocol-build-an-ios-instant-messaging-app-part-2.html).
+
+## Events
+
+The pace of local meetings and sprints is still high within the XMPP community.
+
+pep tells us the story of the "[Sprint in the cold north](https://bouah.net/2019/10/sprint-in-the-cold-north/)", with sauna and crêpes, working on new groupchat bookmarks specification, file transfer interoperability issues, and a future landing page for new XMPP users.
```

Comment:
Do we need to html-encode the letter 'ê'?

Reply:
This shouldn’t be needed anymore in any modern web service, UTF-8 became the de-facto standard and all documents encoded with it will render properly on all user-agents.

Reply:
OK, then I keep it that way? We'll check afterwards, on the newsletter verification, the blog post, and the aggregators.

Reply:
html-encoding is very 1990. I'm showing my age here. It'll be fine just leaving it as-is. :)